pydicom/dataelem.py

```diff
@@ -511,11 +510,10 @@ def _convert_value(self, val):
         except AttributeError:  # not a list
             return self._convert(val)
         else:
-            return MultiValue(lambda x: self._convert(x), val)
+            return MultiValue(self._convert, val)
 
```

Comment:
Yeah, that was an completely unneeded lambda I introduced here. I tend to forget about pickling - we (I) should really make that a habit to test pickling...